src/main/java/io/codeka/gaia/registries/AbstractRegistryApi.kt

```diff
@@ -38,7 +42,7 @@ abstract class AbstractRegistryApi<K: SourceRepository>(val restTemplate: RestTe
 
         val token = user.oAuth2User?.token!!
 
-        return callWithAuth(url, token, registryListFileClass).toList()
+        return callWithAuth(url, token, registryListFileClass)!!.toList()
```

Comment:
we might have an error if the call returns a 404, no ?

Reply:
Yeah that's true.
But it should never happen, as it would mean that the user doesn't exist on github or gitlab.

I'll write this a better way (elvis operator) and add a test to cover it.